src/Ledger/Conway/Specification/Epoch.lagda

```diff
@@ -446,6 +447,21 @@ its results by carrying out each of the following tasks.
 
 \begin{NoConway}
 \begin{figure*}[ht]
+\begin{code}[hide]
+calculatePoolDistr : Snapshot → PoolDistr
+calculatePoolDistr ss =
+    zipWithᵐ (λ c pp → (? , ?) ) sd (poolParameters ss)
```

Comment:
One challenge here is that I need to prove for the first component of the pair that c/totalStake is a number between 0 and 1.

Another challenge is that pool parameters don't seem to have the `poolVRF` field that the spec requires. I need to find out what to use instead.

Reply:
I'm clamping for now.